### PR TITLE
fix(v11): update accordion/button/structured-list/skip-to-content and storybook theme

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Description

fix accordion, button styles, structured-list, skip-to-content and update storybook theme

### Changelog

**New**

- {{new thing}}

**Changed**

- use `convert` to apply `rem`
- get storybook theming to work
- downgrade sass so extend works properly
- update structured-list to add new visually-hidden class to hide radio-buttons
- update skip-to-content to use the new prefix

**Removed**

- {{removed thing}}

### CODE REVIEW

File: packages/carbon-web-components/package.json [hunk 1]
```diff
@@ -192,7 +192,7 @@
     "rollup-plugin-multi-input": "^1.3.1",
     "rollup-plugin-terser": "^7.0.2",
     "rtlcss": "^3.0.0",
-    "sass": "~1.57.0",
+    "sass": "~1.40.0",
```
Comment (kennylam) [inline packages/carbon-web-components/package.json:195]: @ariellalgilmore did you mean to bump `sass` down?

Comment (ariellalgilmore) [reply packages/carbon-web-components/package.json:195]: @kennylam yeah `sass` has an error with `@extend` causing the styles to break. but actually they just released 1.58 which  should hopefully fix it. https://github.com/sass/dart-sass/releases

`Fix a bug in @extend logic where certain selectors with three or more combinators were incorrectly considered superselectors of similar selectors with fewer combinators, causing them to be incorrectly trimmed from the output.`

Comment (kennylam) [reply packages/carbon-web-components/package.json:195]: Gotcha. Another thing we could try is just to keep in sync with the v11 sass version (`1.51.0`). But as long as it's working now then all good.

File: packages/carbon-web-components/src/components/accordion/accordion.scss [hunk 2]
```diff
@@ -10,6 +10,7 @@ $css--plex: true !default;
 @use '@carbon/styles/scss/config' as *;
 @use '@carbon/styles/scss/spacing' as *;
 @use '@carbon/styles/scss/type' as *;
+@use '@carbon/styles/scss/utilities/convert';
```
Comment (emyarod) [inline packages/carbon-web-components/src/components/accordion/accordion.scss]:
```suggestion
@use '@carbon/styles/scss/utilities/convert' as *;
```

File: packages/carbon-web-components/src/components/accordion/accordion.scss [hunk 3]
```diff
@@ -31,14 +32,14 @@ $css--plex: true !default;
 
   &[size='sm'] {
     .#{$prefix}--accordion__heading {
-      min-height: rem(32px);
-      padding: rem(5px) 0;
+      min-height: convert.rem(32px);
+      padding: convert.rem(5px) 0;
```
Comment (emyarod) [inline packages/carbon-web-components/src/components/accordion/accordion.scss]:
```suggestion
      min-height: rem(32px);
      padding: rem(5px) 0;
```

File: packages/carbon-web-components/src/components/accordion/accordion.scss [hunk 4]
```diff
@@ -31,14 +32,14 @@ $css--plex: true !default;
...
     }
   }
 
   &[size='xl'] {
     .#{$prefix}--accordion__heading {
-      min-height: rem(48px);
+      min-height: convert.rem(48px);
```
Comment (emyarod) [inline packages/carbon-web-components/src/components/accordion/accordion.scss]:
```suggestion
      min-height: rem(48px);
```

File: packages/carbon-web-components/src/components/button/button.scss [hunk 5]
```diff
@@ -8,12 +8,13 @@
 $css--plex: true !default;
 
 @use '@carbon/styles/scss/config' as *;
+@use '@carbon/styles/scss/utilities/convert';
```
Comment (emyarod) [inline packages/carbon-web-components/src/components/button/button.scss]:
```suggestion
@use '@carbon/styles/scss/utilities/convert' as *;
```

File: packages/carbon-web-components/src/components/button/button.scss [hunk 6]
```diff
@@ -8,12 +8,13 @@
...
 @use '@carbon/styles/scss/components/button';
 
 :host(#{$prefix}-btn),
 :host(#{$prefix}-modal-footer-button) {
   display: inline-flex;
-  max-width: rem(320px);
+  max-width: convert.rem(320px);
```
Comment (emyarod) [inline packages/carbon-web-components/src/components/button/button.scss]:
```suggestion
  max-width: rem(320px);
```

File: packages/carbon-web-components/src/components/button/button.scss [hunk 7]
```diff
@@ -24,17 +25,17 @@ $css--plex: true !default;
   ::slotted([slot='icon']) {
     fill: currentColor;
     position: absolute;
-    right: rem(16px);
+    right: convert.rem(16px);
```
Comment (emyarod) [inline packages/carbon-web-components/src/components/button/button.scss]:
```suggestion
    right: rem(16px);
```

File: packages/carbon-web-components/src/components/button/button.scss [hunk 8]
```diff
@@ -24,17 +25,17 @@ $css--plex: true !default;
...
     flex-shrink: 0;
   }
 
   &[isExpressive] ::slotted([slot='icon']) {
-    width: rem(20px);
-    height: rem(20px);
+    width: convert.rem(20px);
+    height: convert.rem(20px);
```
Comment (emyarod) [inline packages/carbon-web-components/src/components/button/button.scss]:
```suggestion
    width: rem(20px);
    height: rem(20px);
```

File: packages/carbon-web-components/src/components/button/button.scss [hunk 9]
```diff
@@ -24,17 +25,17 @@ $css--plex: true !default;
...
   }
 
   &[icon-layout='condensed'] .#{$prefix}--btn {
-    padding-right: rem(39px);
+    padding-right: convert.rem(39px);
```
Comment (emyarod) [inline packages/carbon-web-components/src/components/button/button.scss]:
```suggestion
    padding-right: rem(39px);
```

File: packages/carbon-web-components/src/components/button/button.scss [hunk 10]
```diff
@@ -45,11 +46,11 @@ $css--plex: true !default;
   &[kind='danger-ghost'] {
     ::slotted([slot='icon']) {
       position: static;
-      margin-left: rem(8px);
+      margin-left: convert.rem(8px);
```
Comment (emyarod) [inline packages/carbon-web-components/src/components/button/button.scss]:
```suggestion
      margin-left: rem(8px);
```

File: packages/carbon-web-components/src/components/button/button.scss [hunk 11]
```diff
@@ -45,11 +46,11 @@ $css--plex: true !default;
...
     }
 
     &[icon-layout='condensed'] .#{$prefix}--btn {
-      padding-right: rem(16px);
+      padding-right: convert.rem(16px);
```
Comment (emyarod) [inline packages/carbon-web-components/src/components/button/button.scss]:
```suggestion
      padding-right: rem(16px);
```

**Review by emyarod — COMMENTED** [review]
I think we can import into the global namespace so we don't have to update each line that uses the `rem` function right?

File: packages/carbon-web-components/.storybook/_container.scss [hunk 12]
```diff
@@ -7,6 +7,7 @@
 // LICENSE file in the root directory of this source tree.
 //
 
+@use '@carbon/styles';
```
Comment (emyarod) [inline packages/carbon-web-components/.storybook/_container.scss]:
```suggestion
```

this import isn't needed right?

**Review by emyarod — COMMENTED** [review]
structured list and skip to content look good to me! just a few discrepancies I saw but I believe any that aren't addressed here can be addressed as part of the larger sync effort

accordion item font mismatch

image

* need to remove `field` button size and add `2xl`
* icon button requires tooltip

image

* `danger-tertiary` and `danger-ghost` need updates

**Review by emyarod — APPROVED** [review]
looks good to me!

remaining action items for the larger sync effort have been added to the corresponding component issues

**Pull request merged**